### DISCUSSION TITLE
inconsistency between "up" metric and targets page

### DISCUSSION THREAD

**Discussion by dudicoco** [prometheus/prometheus#10361] [Q&A]
The `up` metric is constantly reporting `0` for a couple of targets (pods in our cluster), however, when checking the same targets within the UI's `targets` page, they appear as `up` with no issues.

This only happens with two out of hundreds of targets.

* Prometheus version:

	`2.32.1`

**Comment by roidelapluie** [comment]
Can you tell us what are the other values? scrape_samples_post_metric_relabeling, scrape_samples_scraped, etc ? Can you raise log level the debug to see of there is more information?

**Reply by dudicoco** [reply]
Hi @roidelapluie, the value for `scrape_samples_post_metric_relabeling` and  `scrape_samples_scraped ` for these targets is 170, way below our configured `sample_limit`.

I've enabled debug logs and I don't see any information regarding these targets.

**Reply by roidelapluie** [reply]
Do you have screenshots ? It could help.

**Reply by dudicoco** [reply]
@roidelapluie here are the screenshots:
![Screen Shot 2022-03-16 at 14 47 56](https://user-images.githubusercontent.com/51051860/158593905-211a5296-2b91-40f6-80f0-6be511cabcca.png)

![Screen Shot 2022-03-16 at 14 49 51](https://user-images.githubusercontent.com/51051860/158595296-dcbccfc6-5450-4f80-8a74-463265561f26.jpeg)
